master/internal/db/postgres.go

```diff
@@ -82,7 +82,9 @@ func PaginateBun(
 		direction = SortDirectionAsc
 	}
 	orderExp := fmt.Sprintf("%s %s", orderColumn, direction)
-	query = query.Order(orderExp)
+
+	// needed this in order to order by e.g. metrics->>'loss'
+	query = query.OrderExpr(orderExp)
```

Comment:
this is unsafe, input would need to be sanitized, maybe should add a boolean flag and have safe be default?